File: samples/server/petstore/java-play-framework-api-package-override/app/com/puppies/store/apis/ApiDocController.java
package com.puppies.store.apis;

import .inject.*;
import play.mvc.*;

public class ApiDocController extends Controller {

    @Inject
    private ApiDocController() {
    }

    public Result api() {
        return redirect("/assets/lib/swagger-ui/index.html?url=/assets/openapi.json");
    }
}

File: samples/server/petstore/scala-play-server/app/api/ApiDocController.scala
package api

import javax.inject.{Inject, Singleton}
import play.api.mvc._

@Singleton
class ApiDocController @Inject()(cc: ControllerComponents) extends AbstractController(cc) {
  def api: Action[AnyContent] = Action {
    Redirect("/assets/lib/swagger-ui/index.html?url=/assets/openapi.json")
  }
}
